web/Views/Admin/_MatchListTable.cshtml

```diff
@@ -62,6 +62,10 @@
                         <i class="bi me-1 @GetPreferenceIcon(!tenant.NotInEET)"></i>
                         In EET (Employment, Education, or Training)
                     </div>
+                    <div class="@GetPreferenceFont(tenant.NoGuarantor)">
+                        <i class="bi me-1 @GetPreferenceIcon(tenant.NoGuarantor)"></i>
+                        Without rent guarantor

```

Comment:
Can we make this 'Has rent guarantor' instead? I think that's the default assumption, and the double negative (⛔ Without rent guarantor) is confusing